### ISSUE TITLE
Website "Make it yours" panel does not include font settings

### ISSUE THREAD

**Issue by QWu4xYV** [cschroeter/park-ui#267]
Thanks so much for making this library! Just wanted to point out something that seemed misleading:

The "Copy Config" button in the "Make it yours" panel gives the impression that all the settings set above will be copied, but the font settings are not.

More generally, the Park UI documentation doesn't seem to have guidance around setting fonts; I'd recommend at minimum linking to Panda's ["Custom Font" documentation](https://panda-css.com/docs/guides/fonts).

**Comment by sourcecodes2** [comment]
[Temporary fix](https://github.com/cschroeter/park-ui/issues/261#issuecomment-2046656551)

**Comment by cschroeter** [comment]
Yeah fonts are tricky as depending on your framework setup can be different.
